doc/API/centreon-api-v2.yaml

```
@@ -2252,6 +2280,50 @@ components:
         type: integer
         format: int64
         example: 1
+    ServerAddress:
+      in: query
+      name: address
+      required: true
+      schema:
+        type: string
+        example: "1.1.1.2"
+      description: "Address of the server. IPv4, IPv6 or DNS"
+    ServerName:
+      in: query
+      name: server_name
+      required: true
+      schema:
+        type: string
+        example: "my_poller"
+      description: "Name of the server"
+    ServerType:
+      in: query
+      name: server_type
+      required: true
+      schema:
+        type: integer
```

@kduret , the documentaton needs an update ;) 
```suggestion
        type: string
```